src/lib/plugins/log.js

```diff
@@ -42,7 +40,11 @@ module.exports.server = function (serv, settings) {
   }
 
   serv.err = message => {
-    serv.log('[' + colors.red('ERR') + ']: ' + message)
+    serv.log('[' + colors.red('ERROR') + ']: ' + message)
+  }
+
+  serv.warn = message => {
```

Comment:
Add it to the doc

Is is useful? If yes, use it at some places in flying squid

Reply:
done